components/Datasets/DescribeResource.vue

```diff
@@ -355,8 +361,8 @@ const setFiles = (files: Array<File>) => {
   }
 }
 
-const { data: extensions } = await useAPI<Array<string>>('/api/1/datasets/extensions/')
-const { data: schemas } = await useAPI<SchemaResponseData>('/api/1/datasets/schemas/')
+const { data: extensions, status } = await useAPI<Array<string>>('/api/1/datasets/extensions/', { lazy: true })
```

Comment:
These changes and the new loading blocks prevents the modal showing with an empty body while the component is loading async data